pages/gallery.html

```diff
@@ -1,12 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="/styles/styles.css">
+	<link rel="stylesheet" href="/styles/gallery.css">
+	<title>Gallery</title>
 </head>
+
 <body>
-  
+	<nav class="nav">
+		<h1>Janice and Ericka's Grey's Anatomy Fan Page!</h1>
+		<h3>Gallery</h3>
```

Comment:
For semantic HTML, I'd expect there to be `h2` used before `h3`. If you need it to be `h3` in order to have a better font size, you should use `h2` still and use CSS to change the text size.

This goes throughout your project